### ISSUE TITLE
Resume from checkpoint

### ISSUE THREAD

**Issue by JinSeoungwoo** [erfanzar/EasyDeL#46]
When I load model from a checkpoint and try to continue training at the previous learning rate, the loss increases sharply after a small step, so I have to warm up. Is this normal?

**Comment by erfanzar** [comment]
hi 
in case that you have changed the training configs such as optimizer , learning rate or dataset yes this is normal
